### PR TITLE
Add Joomla!

### PR DESCRIPTION
**Pull request by anibalsanchez**
**https://github.com/anibalsanchez/awesome-joomla**

**
The list has a set of actual links that Joomla! volunteers use on a daily basis to collaborate with the project. Joomla! is a community project with more than 10 years of history and the list of awesome resources will be focused on the best resources available today. For instance, the upcoming Joomla! 4 or the current Google Summer of Code (where I'm mentoring students).

This list will certainly growth with time and when more volunteers start to know about of its existence. 
**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Has the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -406,6 +406,7 @@
 - [Wagtail](https://github.com/springload/awesome-wagtail) - Django CMS focused on flexibility and user experience.
 - [Textpattern](https://github.com/drmonkeyninja/awesome-textpattern) - Lightweight PHP-based CMS.
 - [Drupal](https://github.com/nirgn975/awesome-drupal) - Extensible PHP-based CMS.
+- [Joomla!](https://github.com/anibalsanchez/awesome-joomla) - The CMS Trusted By Millions for their Websites.
```
Comment (sindresorhus) [inline readme.md]: The description should be succinct description of what Joomla is, not PR talk.

**Comment by sindresorhus** [comment]
> No templates. No extensions.

I don't see the point of this restriction. You could list the most awesome ones.

---

I don't think "Leadership" section is of much interest to users.

---

The "Community" section should use a list and not a table. Tables provides no benefit for the current usage.

---

The items in `Experts on Twitter` needs a description on who they are and why a visitor should care and follow them.

---

Not a lot of actual content here. Missing blog posts, videos, tutorials, libraries, themes, etc. The whole list in its current form feels more like a promotional page than an actual resource for users.

**Comment by anibalsanchez** [comment]
Thank you very much for your review. I'm still grasping the style to create the list.

Right now, the list is being commented in the Joomla! glip channel. More contributions are coming. :-D

Please, give me a couple of weeks to add your suggestions.

A few notes:

* **... not PR talk** Ok, I'll purge the marketing sentences, and add more info like standard references. (Sorry, if my English is limited... I'm going to add a reference from Wikipedia)
* **No templates. No extensions.** There are around 8,000 extensions on JED and I can't even count the top templates. So, to keep the list clean, we have to avoid naming extensions or templates.... however, we can include Frameworks for extensions or templates. I'll add them.
* **Leadership**, Ok, I'll move it to the end. In my opinion, the community is an important point of Joomla!
* **Community**, Ok, I'll remove the table
* **Experts on Twitter**, Ok, I'll complete the list. I've just included it today.

**Comment by sindresorhus** [comment]
> There are around 8,000 extensions on JED and I can't even count the top templates.

There are 475000 modules on https://www.npmjs.com/, but I still managed to pick the best ones in https://github.com/sindresorhus/awesome-nodejs That's the point of Awesome - showing the best of something. You could pick the best 20 or 50 extensions and themes.

**Comment by anibalsanchez** [comment]
Okok, I'll do my best to pick the extensions that are the best ones.

**Comment by sindresorhus** [comment]
Closing for lack of activity.

**Pull request closed without merging**